src/screens/EtherspotDeploymentInterjection/EtherspotDeploymentInterjection.js

```diff
@@ -127,6 +179,22 @@ function EtherspotDeploymentInterjection() {
               btnTextStyle={styles.btnTextStyle}
             />
           </ButtonContainer>
+          {!gasInfo.isFetched && <SpinnerWrapper><Spinner size={20} trackWidth={2} /></SpinnerWrapper>}
+          {!!deploymentFee && (
+            <Tooltip
+              body={deploymentFee.tokenValue}
+              isVisible={showDeploymentFeeInToken}
+              positionOnBottom={false}
+            >
+              <TouchableOpacity
+                hitSlop={hitSlop10}
+                activeOpacity={1}
+                onPress={() => setShowDeploymentFeeInToken(!showDeploymentFeeInToken)}
+              >
+                <FeeText>{t('cost', { cost: deploymentFee.fiatValue })}</FeeText>
+              </TouchableOpacity>
+            </Tooltip>
+          )}
           <BottomText>{t('bottomText')}</BottomText>
```

Comment:
Bottom text should using native asset symbol, instead of hardcoded ETH atm: 
![image](https://user-images.githubusercontent.com/6368606/129889977-2db405ed-7993-44b8-9375-6caddbb95c3d.png)

Reply:
This one will be removed per chat with Dmitry.